client/my-sites/exporter/select.jsx

```diff
@@ -2,23 +2,91 @@
  * External dependencies
  */
 import React, { Component } from 'react';
+import { get } from 'lodash/object';
+import { connect } from 'react-redux';
 
 /**
  * Internal dependencies
  */
 import FormSelect from 'components/forms/form-select';
+import {
+	getPostTypeFieldOptions,
+	getPostTypeFieldValue,
+} from 'state/site-settings/exporter/selectors';
+import {
+	setPostTypeFieldValue,
+} from 'state/site-settings/exporter/actions';
+import i18n from 'i18n-calypso';
 
 export default class Select extends Component {
 	render() {
-		const options = this.props.options.map( ( option, i ) => {
+		const {
+			isEnabled,
+			isError,
+			postType,
+			shouldShowPlaceholders,
+			value,
+			fieldName,
+		} = this.props;
+
+		const fieldsForPostType = get( {
+			post: [ 'author', 'status', 'start_date', 'end_date', 'category' ],
+			page: [ 'author', 'status', 'start_date', 'end_date' ],
+		}, postType, [] );
+
+		const label = get( {
+			author: i18n.translate( 'Author…' ),
+			status: i18n.translate( 'Status…' ),
+			start_date: i18n.translate( 'Start Date…' ),
+			end_date: i18n.translate( 'End Date…' ),
+			category: i18n.translate( 'Category…' ),
```

Comment:
Also see: https://github.com/Automattic/i18n-calypso/blob/master/README.md#localize for more context.